type error when using i18next v20.3.4

## 🐛 Bug Report

After updating i18next from 20.3.3 to 20.3.4 typescript won't build the frontend code when i18next-http-backend 1.2.1 is used.
```
❯ npx tsc
node_modules/i18next-http-backend/index.d.ts:108:3 - error TS2416: Property 'create' in type 'I18NextHttpBackend' is not assignable to the same property in base type 'BackendModule<BackendOptions>'.
  Type '((languages: string | string[], namespace: string, key: string, fallbackValue: string) => void) | undefined' is not assignable to type '((languages: readonly string[], namespace: string, key: string, fallbackValue: string) => void) | undefined'.
    Type '(languages: string | string[], namespace: string, key: string, fallbackValue: string) => void' is not assignable to type '(languages: readonly string[], namespace: string, key: string, fallbackValue: string) => void'.
      Types of parameters 'languages' and 'languages' are incompatible.
        Type 'readonly string[]' is not assignable to type 'string | string[]'.
          The type 'readonly string[]' is 'readonly' and cannot be assigned to the mutable type 'string[]'.

108   create?(
      ~~~~~~


Found 1 error.
```

## To Reproduce

Install i18next 20.3.4 and i18next-http-backend 1.2.1

```ts
import i18next from 'i18next';
import I18NextHttpBackend from 'i18next-http-backend';
import { initReactI18next } from 'react-i18next';
import { baseUrl, isDev } from './config';

i18next
  .use(I18NextHttpBackend)
  .use(initReactI18next)
  .init({
    fallbackLng: 'en',
    debug: isDev,
    load: 'languageOnly',
    backend: {
      loadPath: `${baseUrl}/locales/{{lng}}.json`,
    },
    interpolation: {
      escapeValue: false, // not needed for react as it escapes by default
    },
  });

export default i18next;
```

## Expected behavior

Typescript builds this.

## Your Environment

- *runtime version*: node v14
- *i18next version*: 20.3.4
- *os*: Linux

--- a/index.d.ts
+++ b/index.d.ts
@@ -106,7 +106,7 @@ export default class I18NextHttpBackend
     namespaces?: string | string[]
   ): void;
   create?(
-    languages: string | string[],
+    languages: string[],
     namespace: string,
     key: string,
     fallbackValue: string

